website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -90,6 +92,85 @@ metrics:
 
 </File>
 
+<VersionBlock firstVersion="1.2">
+
+### Available properties
+
+| Field       | Description                                                 | Example                         | Required? |
+|-------------|-------------------------------------------------------------|---------------------------------|-----------|
+| name        | A unique identifier for the metric                          | new_customers                   | yes       |
+| model       | The dbt model that powers this metric                       | dim_customers                   |     yes (no for `expression` metrics)    |
```

Comment:
[Blocking] From a quick visual inspection, I think this is the only change in the table between v1.0/v1.1 &rarr; v1.2:
- `model` + `Required?`: `yes (no for expression metrics)`, instead of `yes` (= always) 

Is that right?

I think our `VersionBlock` logic might be fancy enough to make this work:
```suggestion
| model       | The dbt model that powers this metric                       | dim_customers                   | <VersionBlock firstVersion="1.2">yes (no for `expression` metrics)</VersionBlock><VersionBlock lastVersion="1.1">yes</VersionBlock> |
```
I tried that locally and it seemed to do the trick. Then we don't need to duplicate the full table contents for different docs versions.